test/routes/src/server/routes/t_webhook.js

```diff
@@ -393,5 +404,148 @@ describe('The WebHook API endpoint', () => {
           });
       });
     });
+
+    describe('with a good signature from Launchpad', () => {
+      beforeEach(async () => {
+        await db.model('GitHubUser').query('truncate').fetch();
+        await db.model('GitHubUser')
+          .forge({
+            github_id: 1,
+            login: 'anowner',
+            last_login_at: new Date()
+          })
+          .save();
+      });
+
+      describe('if store_upload_status is not Uploaded', () => {
```

Comment:
As a convention we tend to be using `describe` only on top level (when we say we 'describe' a 'thing' that is being tested) and for different variants/scenarios we use `context` (when we say how a thing should work in specific case).